mongoDB.go

```diff
@@ -113,6 +117,47 @@ func mapUniqueRecords(records *[]RecordLink) map[string]aggregatedRecord {
 	return recordMap
 }
 
+// Attempts to read the geometry from the Postgres table and update to Mongo
+func findElementPoints(s *Server, session *mgo.Session, record *RecordLink, tableMap map[int]TableMeta) {
+	// Only update if no lat or lon information stored.
+	if record.Lat < 0.1 && record.Lat > -0.1 && record.Lon < 0.1 && record.Lon > -0.1 {
+		var newLat, newLon float64
+		tableName := tableMap[record.TableID].Table
+		fieldName := tableMap[record.TableID].KeyField
```

Comment:
By the time that this code is executed it is guaranteed that the element is present in the map.